draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -2716,6 +2716,11 @@ Host: www.example.org
    architecture, there are no inherent limits to the extent an intermediary
    can enhance (or interfere) with either direction of the stream.
 </t>
+<t>
+   Intermediaries are expected to forward messages even when protocol elements
+   such as methods, status codes, and fields are not recognized, as doing so
+   preserves the extensibility of the protocol.
```

Comment:
```suggestion
   Intermediaries are expected to forward messages even when protocol elements
   are not recognized (e.g., new methods, status codes, or field names), since that
   preserves extensibility for downstream recipients.
```